theme/templates/resource-landing-page/modals.html

```diff
@@ -777,40 +777,6 @@ <h4 class="modal-title" id="extra_meta_title"> Add/Edit Extended Metadata </h4>
     </div>
 </div>
 
-<!-- Modal -->
-{%  block modal_add_file %}
-<div class="modal fade" id="add-file-dialog" tabindex="-1" role="dialog" aria-labelledby="add-file"
-     aria-hidden="true">
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <form action="/hsapi/_internal/{{ cm.short_id }}/add-file-to-resource/" method="POST"
-                  enctype="multipart/form-data">
-
-                <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                    <h4 class="modal-title" id="add-file">Add file to resource</h4>
-                </div>
-                <div class="modal-body">
-                    {% csrf_token %}
-                    <div class="form-group">
-                        <label for="add-file-input">Select files:</label>
-                        <input type="file" name="files" id="add-file-input" multiple/>
-                    </div>
-                    <div id="file-types">Any file type can be uploaded.</div>
-                    <div id="file-multiple">Multiple file upload is allowed.</div>
-                    <br>
-                    <h3 style="color: red" id="file-type-error"></h3>
-                </div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                    <button type="submit" class="btn btn-primary">Save changes</button>
-                </div>
-            </form>
-        </div>
-    </div>
-</div>
-{%  endblock %}
-
```

Comment:
@mauriel133 OK, now I see this add file block is included in genericresource.html as well, so the duplicate one in this modals.html can be removed safely. a +1 from me.